src/web/components/SharingPermission/ParticipantItem.tsx

```diff
@@ -61,6 +78,18 @@ export function ParticipantItem({ site, onClick, checked }: ParticipantItemProps
       </td>
       <ParticipantItemSimple site={site} />
       {isSharingParticipant(site) && <td>{formatSourceColumn(site.addedBy)}</td>}
+      {onDelete && (
```

Comment:
why is this being conditionally shown?  shouldn't there always be an onDelete prop?

Reply:
If this condition is not there, the delete button will show up on the dropdown when adding permissions because the same `ParticipantTable` component is used for the dropdown when searching and when displaying which permissions have been added